spec/features/search_page_spec.rb

```diff
@@ -63,4 +63,80 @@
 
   end
 
+  scenario 'Match location query against available locations', :js => true do
+
+    first_dataset = DatasetBuilder.new
+                      .with_title('Wellington Dataset')
+                      .with_location('Wellington')
+                      .build
+
+    second_dataset = DatasetBuilder.new
+                       .with_title('Welding Dataset')
+                       .with_location('Welding')
+                       .build
+
+    index([first_dataset, second_dataset])
+
+    visit('/search')
+
+    assert_data_set_length_is(2)
+
+    page.driver.execute_script %Q{ $('#location').focus().typeahead('val', 'We') }
+    page.driver.execute_script %Q{ $('#location').focus().typeahead('open') }
+
+    expect(page).to have_selector('.tt-selectable', count: 2)
+    expect(page).to have_css('.tt-selectable', text: 'Wellington')
+    expect(page).to have_css('.tt-selectable', text: 'Welding')
+
+    page.driver.execute_script %Q{ $('#location').focus().typeahead('val', 'Wellington') }
```

Comment:
I was not able to get capybara to select the item from the dropdown. Had to use Jquery to set typeahead's value manually  ... hack ??

Reply:
Might be a case of Capybara needing to wait until the typeahead value shows up?

Reply:
Ok - will add a sleep and see if this works

Reply:
Hey @laurentqro - no luck. Capybara fills in the field ok but for some reason this does not seem to fire an event (which ultimately gets picked up by Typeahead)